bitnami/redis/Chart.yaml

```
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.8.9
+version: 16.8.10
```

Thanks for the PR @vijay2308! Could you please modify the version and bump its minor. Even though it is small, we are adding a new feature.

```suggestion
version: 16.9.0
```

---

done!